### PR TITLE
Fixes wire cutting issues.

### PR DESCRIPTION
**Pull request by DominikPanic**
Fixes: #1543

### CODE REVIEW

File: code/datums/wires/_wires.dm [hunk 1]
```diff
@@ -173,6 +180,14 @@
 /datum/wires/proc/pulse(wire, user)
 	if(is_cut(wire))
 		return
+
+	if(usr.mind?.cm_skills && usr.mind.cm_skills.engineer < SKILL_ENGINEER_ENGI)
```
Comment (spookydonut) [inline code/datums/wires/_wires.dm]: this should be above is_cut check since wires could be cut while someone is fumbling

**Comment by spookydonut** [comment]
You also haven't explained the issue in the PR description.

File: code/datums/wires/_wires.dm (hunk 1 above)
Comment (DominikPanic) [reply code/datums/wires/_wires.dm]: That was done on purpose to reduce the frustration for the users - though I forgot the is_cut check in the do-after.

**Comment by DominikPanic** [comment]
> 
> 
> You also haven't explained the issue in the PR description.

The linked issue explains it.

**Comment by spookydonut** [comment]
Needs a valid description and changelog entry

**Comment by Rohesie** [comment]
The changelog for fixes is kind of optional, though welcome.
For the rest this PR looks ok and not invalid?

**Pull request merged**